### PR TITLE
Setup Author Envionment

### PR DESCRIPTION
**Pull request by jonnyshaw89**
As part of the work for setting up a production environment for EQ Author, this PR is being raised to prove setting up an isolated environment for Author with it's own working implementation of survey runner running in a separate VPC and it's own ECS cluster.

## Depends On
 - https://github.com/ONSdigital/eq-terraform/pull/144
 - https://github.com/ONSdigital/eq-terraform-ecs/pull/26
 - https://github.com/ONSdigital/eq-ecs-deploy/pull/21

### CODE REVIEW

**Comment by samiwel** [comment]
Tested spinning it up and tearing it down. Seems to work fine. Was able to sign in, create a simple questionnaire and preview in runner. 🤷‍♂️ LGTM

**Review by samiwel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+
+```
+env="" # your name (Do not use underscore)
+
+# ask somebody on the team to send these values to you
+aws_account_id=""
+aws_assume_role_arn=""
+ons_access_ips=""
+certificate_arn=""
+slack_webhook_path=""
```
Comment (ajmaddaford) [inline README.md:30]: Also `author_firebase_project_id` and `author_firebase_api_key`

Comment (ajmaddaford) [reply README.md:30]: These used to be defaulted in eq-terraform to something made up so that they weren't needed in the default setup of auth being off.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jonnyshaw89) [reply README.md:30]: Updated

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**